core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/delete-answer-group-modal.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for Delete Answer Group Modal.
+ */
+
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { DeleteAnswerGroupModalComponent } from './delete-answer-group-modal.component';
+
+describe('Delete Answer Group Modal Component', () => {
+  let fixture: ComponentFixture<DeleteAnswerGroupModalComponent>;
+  let componentInstance: DeleteAnswerGroupModalComponent;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DeleteAnswerGroupModalComponent
+      ],
+      providers: [
+        NgbActiveModal
+      ]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DeleteAnswerGroupModalComponent);
+    componentInstance = fixture.componentInstance;
+  });
+
+  it('should create', () => {
```

Comment:
Please add a comment on why there are no other frontend tests in this file.

Reply:
Done, Please check comment, is it okay?

Reply:
try and mention that it inherits the `ngbActiveModal` class and it doesn't add any additional functionalities to it

Reply:
Done now